boot/freeldr/freeldr/lib/mm/meminit.c

```diff
@@ -262,6 +262,7 @@ MmCheckFreeldrImageFile(VOID)
         (FileHeader->NumberOfSymbols != 0) ||       //    ""      ""
         (FileHeader->SizeOfOptionalHeader != sizeof(IMAGE_OPTIONAL_HEADER)))
     {
+    #ifndef UEFIBOOT
```

Comment:
This is disabled in UEFI because of which part(s) that may change? I suppose: number of sections, and what else?

Reply:
In hindsight going into this function at all doesn't make sense. We don't even use FrLdrImageSize for UEFI and that's the only logic happening here.

Should i just #ifndef UEFIBOOT the call inside freeldr.c instead of doing this preprocessor mess here?